app/screens/settings/notification_mention/mention_settings.tsx

```diff
@@ -33,10 +33,11 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
     return {
         input: {
             color: theme.centerChannelColor,
-            height: 150,
-            paddingHorizontal: 15,
             ...typography('Body', 100, 'Regular'),
         },
+        inputContainer: {
+            height: 128,
```

Comment:
not sure why you reduced the size of this, also the label seems to be in the wrong place as is aligned to the border.

You can use The Edit Channel screen as reference as that one is rendering correctly

Reply:
![Screenshot 2022-11-21 at 10 18 53](https://user-images.githubusercontent.com/17684951/203025809-3b9c0118-2854-4491-bcd1-e8cbfe0b704a.png)
![Screenshot 2022-11-21 at 10 19 29](https://user-images.githubusercontent.com/17684951/203025823-89645b33-0f4b-4bef-97da-71da657d5c1f.png)

Answering to why the dimension was reduced, the reason was because the height provided was not being applied properly, for example in this specific scenario the provided height was of 150, but actually the applied height would result in 102(I just tested this through the on Layout), I can't remember why I went with 128 back then (it's been a while) but with the current approach its much easier to go into the Figma files and provide the proper height values

Reply:
As for the label position, I'm having an hard time figuring out what's wrong about it's position

Gekidou branch:
![Screenshot 2022-11-21 at 10 28 17](https://user-images.githubusercontent.com/17684951/203027506-714d0d38-8782-4dc1-a4ea-2d42ac43afc4.png)

feat/MM-46562-toggle-password-visibility branch:
![Screenshot 2022-11-21 at 10 29 07](https://user-images.githubusercontent.com/17684951/203027692-9e97ebf6-3b78-4368-881c-627a51180b0e.png)